### PR TITLE
initialize sprintboot

### LINKED ISSUE #11: initialize springboot

### PR DESCRIPTION
**Pull request by Jor-Moreno**
This PR initializes spring boot

Currently, the application uses H2 for database memory. This is temporary and is configured under src\main\resources\application.properties. 
Followup would be changing to MySQL or PostgreSQL depending on the database host. 

Part of issue #11

### CODE REVIEW

File: src/test/java/com/jorge/whitelist/WhitelistApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.jorge.whitelist;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class WhitelistApplicationTests {
```
Comment (Bedolla233) [inline src/test/java/com/jorge/whitelist/WhitelistApplicationTests.java:7]: Correct me if I am wrong, does this test always pass as long as the application loads properly?

**Review by Bedolla233 — APPROVED** [review]
All the code is properly formatted and has passing test. Will approve, just maybe make sure to add a comment about the website we had found that gives us a guide on starting a springboot app,

File: src/test/java/com/jorge/whitelist/WhitelistApplicationTests.java (hunk 1 above)
Comment (Jor-Moreno) [reply src/test/java/com/jorge/whitelist/WhitelistApplicationTests.java:7]: Yes, since we do not have anything connected it is empty for now.

**Comment by Jor-Moreno** [comment]
> All the code is properly formatted and has passing test. Will approve, just maybe make sure to add a comment about the website we had found that gives us a guide on starting a springboot app,

Can be done as a followup with the docker, since we will be using docker to run our environment.

**Review by Noel-Hann — APPROVED** [review]
I took a look at it, it initializes and passes the test

**Pull request merged**